protos/perfetto/trace/perfetto_trace.proto

```diff
@@ -16514,6 +16514,10 @@ message CounterDescriptor {
 
   // TODO(eseckler): Support arguments describing the counter (?).
   // repeated DebugAnnotation debug_annotations;
+
+  // All counter tracks with the same |y_axis_share_key| and the same parent
```

Comment:
Make sure to run tools/gen_all out/linux_clang_release once you make other changes.